### PR TITLE
[#808] non root pipeline jobs

### LINKED ISSUE #808: Consider dockerized pipeline steps execution from non-priveleged user
Some pipeline steps are executed in docker container which is working from root user. We should think of reducing privileges due to security reasons and because of potential issues with build artifacts cleanup.

### PR DESCRIPTION
**Pull request by alexander-semenets**
this fixes #808 
There are also playbooks for jenkins agents configuration in legion-ci repo

!REMINDER:
- jenkins agent AMIs should be updated after merge.

### CODE REVIEW

**Comment by alexander-semenets** [comment]
bot, test all

**Comment by alexander-semenets** [comment]
bot, test all

File: Jenkinsfile [hunk 1]
```diff
@@ -493,7 +493,7 @@ EOL
                 stage('Package and upload helm charts'){
                     steps {
                         script {
-                            docker.image("legion/legion-docker-agent:${Globals.buildVersion}").inside("-v /var/run/docker.sock:/var/run/docker.sock --net host") {
+                            docker.image("legion/legion-docker-agent:${Globals.buildVersion}").inside("-v /var/run/docker.sock:/var/run/docker.sock -u root") {
```
Comment (mcdoker18) [inline Jenkinsfile]: Why did you add root user?
I think it a reason for the failed job https://cc.epm.kharlamov.biz/jenkins/job/Build_Legion_Artifacts/1655/

File: deploy/helms/airflow/templates/mq.yaml [hunk 2]
```diff
@@ -40,7 +40,7 @@ spec:
     spec:
       containers:
       - name: redis
-        image: "redis:3@sha256:7b0a40301bc1567205e6461c5bf94c38e1e1ad0169709e49132cafc47f6b51f3"
+        image: "redis:3"
```
Comment (mcdoker18) [inline deploy/helms/airflow/templates/mq.yaml]: Can you specify precise version?

File: deploy/helms/airflow/templates/db.yaml [hunk 3]
```diff
@@ -38,7 +38,7 @@ spec:
     spec:
       containers:
       - name: postgres
-        image: "postgres:10@sha256:41f013d3153c03345d04ae362b2ce4b60f4a1687f260aafd74f75461e5e5be8d"
+        image: "postgres:10"
```
Comment (mcdoker18) [inline deploy/helms/airflow/templates/db.yaml]: Can you specify precise version?

**Review by mcdoker18 — CHANGES_REQUESTED** [review]

**Comment by alexander-semenets** [comment]
bot, test all

**Review by mcdoker18 — DISMISSED** [review]

**Comment by alexander-semenets** [comment]
bot, test all

**Comment by mcdoker18** [comment]
bot, test all

**Comment by alexander-semenets** [comment]
bot, test all

**Review by mcdoker18 — APPROVED** [review]

**Pull request merged**